front/tests/010-op-route-tab.spec.ts

```diff
@@ -0,0 +1,240 @@
+
+test.beforeAll(async () => {
+  smallInfra = (await getInfra()) as Infra;
+  project = await getProject();
+  study = await getStudy(project.id);
+});
+
+test.beforeEach(async () => {
+  timetable = await postApiRequest(`/api/v2/timetable/`, {
+    electrical_profile_set_id: null,
+  });
+  scenario = await postApiRequest(`/api/v2/projects/${project.id}/studies/${study.id}/scenarios/`, {
+    ...scenarioData,
+    name: `${scenarioData.name} ${uuidv4()}`,
+    study_id: study.id,
+    infra_id: smallInfra.id,
+    timetable_id: timetable.id,
+  });
+});
+
+test.describe('Verifying that all elements in the route tab are loaded correctly', () => {
+  test('should correctly select a route for operational study', async ({ page, browserName }) => {
+    const operationalStudiesPage = new OperationalStudiesPage(page);
+    const scenarioPage = new ScenarioPage(page);
+    const rollingStockSelector = new RollingStockSelectorPage(page);
+
+    // TODO: DROP TSV1: remove this part
+    const homePage = new HomePage(page);
+    await homePage.goToHomePage();
+    const selectedLanguage = await homePage.getOSRDLanguage();
+    await homePage.toggleTSV2();
+
+    // Navigate to the created scenario page
+    await page.goto(
+      `/operational-studies/projects/${project.id}/studies/${study.id}/scenarios/${scenario.id}`
+    );
+
+    // Verify that the infrastructure is correctly loaded
+    await scenarioPage.checkInfraLoaded();
+
+    // Click on add train button
+    await operationalStudiesPage.clickOnAddScenarioTrainBtn();
+
+    // Verify the presence of warnings in Rolling Stock and Route Tab
+    await operationalStudiesPage.verifyTabWarningPresence();
+
+    // Open Rolling Stock Selector, search for the added train, and select it
+    await operationalStudiesPage.openEmptyRollingStockSelector();
+    await rollingStockSelector.searchRollingstock(electricRollingStockName);
+    const electricRollingstockCard = rollingStockSelector.getRollingstockCardByTestID(
+      `rollingstock-${electricRollingStockName}`
+    );
+    await electricRollingstockCard.click();
+    await electricRollingstockCard.locator('button').click();
+
+    // Verify the selected rolling stock name
+    expect(await rollingStockSelector.getSelectedRollingStockName.innerText()).toEqual(
+      electricRollingStockName
+    );
+
+    // Perform pathfinding and verify no selected route
+    await scenarioPage.openTabByDataId('tab-pathfinding');
+    await operationalStudiesPage.verifyNoSelectedRoute(selectedLanguage);
+    await operationalStudiesPage.performPathfindingByTrigram('WS', 'NES', 'MES');
+
+    // Define expected map markers values
+    const expectedMapMarkersValues = ['West_station', 'North_East_station', 'Mid_East_station'];
+
+    // Verify map markers in Chromium browser
+    if (browserName === 'chromium') {
+      await operationalStudiesPage.verifyMapMarkers(...expectedMapMarkersValues);
+    }
```

Comment:
- remove the comment "Define expected map markers" since it is quite obvious
- initialize the const only if we enter the if
- why do we only check en markers in Chromium ? Maybe add a comment to answer this question
```suggestion
    // Verify map markers in Chromium browser
    if (browserName === 'chromium') {
      const expectedMapMarkersValues = ['West_station', 'North_East_station', 'Mid_East_station'];
      await operationalStudiesPage.verifyMapMarkers(...expectedMapMarkersValues);
    }
```